CAM/DoesCamExist.md

```diff
@@ -8,11 +8,10 @@ ns: CAM
 BOOL DOES_CAM_EXIST(Cam cam);
 ```
 
-```
-Returns whether or not the passed camera handle exists.  
-```
+Looks up a camera handle in the current camera pool and returns `True` if the handle is found, otherwise it returns `False`.
```

Comment:
Extreme nit pick, but is there a reason to capitalize True and False?

Reply:
> Extreme nit pick, but is there a reason to capitalize True and False?

Huh, that's a good point. I have always seen it be capitalized, I guess this is to blame?
https://learn.microsoft.com/en-us/dotnet/api/system.boolean.tostring?view=net-7.0

I can make it lowercase.

Reply:
Done & squashed! 🙂